clients/drcpusim/options.cpp

```diff
@@ -127,10 +127,13 @@ droption_t<bool> op_allow_prefetchw(
     "processors, while they do not officially support it, will turn it into a NOP. "
     "As it is commonly seen on Windows, by default drcpusim does not complain about it.");
 
-droption_t<std::string> op_blacklist(
-    DROPTION_SCOPE_CLIENT, "blacklist", IF_WINDOWS_ELSE("ntdll.dll", ""),
-    ":-separated list of libs to ignore.",
-    "The blacklist is a :-separated list of library names for which violations "
+droption_t<std::string> op_blocklist(
+    // We support the legacy name as an alias for compatibility.
+    // We explicitly specity the vector type to avoid ambiguity with iterators
```

Comment:
You mean otherwise it may be considered an `std::pair`?

Reply:
No, it considers the 2 char* to define a start and end iterator.